src/commonMain/kotlin/character/classes/rogue/abilities/SinisterStrike.kt

```diff
@@ -0,0 +1,78 @@
+package character.classes.rogue.abilities
+
+import character.*
+import mechanics.Melee
+import sim.Event
+import sim.SimParticipant
+import data.Constants
+import character.classes.rogue.talents.*
+
+class SinisterStrike : Ability() {
+    companion object {
+        const val name = "Sinister Strike"
+    }
+
+    override val id: Int = 26862
+    override val name: String = Companion.name
+
+    override fun gcdMs(sp: SimParticipant): Int = sp.physicalGcd().toInt()
+
+    override fun resourceType(sp: SimParticipant): Resource.Type = Resource.Type.ENERGY
+    override fun resourceCost(sp: SimParticipant): Double {
+        val improved = sp.character.klass.talents[ImprovedSinisterStrike.name] as ImprovedSinisterStrike?
+        val reduction = improved?.reducedEnergy() ?: 0.0
+        return 45.0 - reduction
+    }
+
+    val bonusDamage = 98.0
+    override fun cast(sp: SimParticipant) {
+        val lethality = sp.character.klass.talents[Lethality.name] as Lethality?
+        val critDmgMultiplier = lethality?.critDamageMultiplier() ?: 1.0
+
+        var increasedDamagePercent = 0.0
+        
+        val aggression = sp.character.klass.talents[Aggression.name] as Aggression?
+        increasedDamagePercent += aggression?.damageIncreasePercent() ?: 0.0
```

Comment:
According to [the old WoW wiki](https://wowwiki-archive.fandom.com/wiki/Sinister_Strike_formulas) they are multiplicative, but in the same wiki they also apply the crit bonus on the whole crit instead of only to the crit bonus dmg and other weird things. Using these formulas does way too much dmg compared to playtesting, so I take anything on there with a huge grain of salt.

From playtesting, this additive approach is almost perfect, it just does a tiny bit too much damage right now (my ingame minhit after 100s of casts are _below_ the minhit of the sim, but just slightly, around 1%)